### PR TITLE
Chore/remove device module

### PR DESCRIPTION
**Pull request by Malak67**
- Removed device module
- Fixed meter reads logic

### CODE REVIEW

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { AggregatedReadDTO } from '@energyweb/origin-drec-api-client';
 import { useTheme } from '@material-ui/core';
+import { getMegawattsFromWatts } from 'utils';
 
 export const useGenerateChartDataset = (reads: AggregatedReadDTO[]) => {
     const theme = useTheme();
-    const currentData = reads.map((read) => read.value);
+    const currentData = reads.map((read) => getMegawattsFromWatts(read.value));
```
Comment (alexworker23) [inline apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts:7]: We have a `PowerFormatter` and `EnergyFormatter` in `@energyweb/origin-ui-utils` package for the same formatting

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts [hunk 2]
```diff
@@ -14,14 +16,18 @@ export const useGenerateChartLabels = ({
...
     const labels: string[] = [];
-    let current = start;
 
-    while (dayjs(current).isBefore(dayjs(end))) {
+    for (
+        let current = start;
+        current < end;
+        current = dayjs(current).add(multiplier, TimeUnitPluralEnum.minutes).toDate()
+    ) {
+        if (labels.includes(dayjs(current).format(format))) {
```
Comment (alexworker23) [inline apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts:25]: way too hacky. needs to be improved

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts (hunk 1 above)
Comment (Malak67) [reply apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts:7]: From what I know PowerFormatter just formats the data as a presentation layer and returns it as a string. PowerFormatter from @energyweb/origin-ui-utils had some issues so I used an internal one. I can check EnergyFormatter and if it makes sense, I will replace this method.

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts (hunk 2 above)
Comment (Malak67) [reply apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts:25]: I know but this issue only happens for Week and Years. For days and months it is working fine. That`s why it is a bit weird. I will try to improve it.

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts (hunk 1 above)
Comment (Malak67) [reply apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartDataset.ts:7]: I checked both methods but they return different values than what I need, and as string.

File: apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts (hunk 2 above)
Comment (Malak67) [reply apps/drec-ui/src/apps/device-group/logic/smartMeter/chart/useGenerateChartLabels.ts]: Found the issue. It was the multiplier minutes. Should work fine now.

**Review by alexworker23 — APPROVED** [review]

**Pull request merged**